.circleci/config.yml

```diff
@@ -107,7 +107,9 @@ workflows:
             tags:
               only: /.*/
       - release-bintray-publish:
-          context: hypertrace-publishing
+          context:
+            - hypertrace-publishing
```

Comment:
do we still need the old context? Also, are there plans on moving this repo off circleci? i think it's the last one preventing us from completely disconnecting cci.

Reply:
This context has `DOCKER_USERNAME` and `DOCKER_PASSWORD`. Not sure if we need `docker login` here.

will move this repo to Github Actions as part of another PR (maybe next week).

Reply:
I see it's setting up and logging into docker, but I can't imagine why. Guessing that was just copy pasted from a job building docker images - this one isn't, right?

Reply:
This repo also builds docker image: https://github.com/hypertrace/javaagent/blob/main/Makefile#L22-L31

It is used for copying java agent to container filesystem before starting the java process using a mutating webhook.

Reply:
I dare not ask why we're using make here..